workflows/models.py

```diff
@@ -1298,7 +1354,10 @@ def get_or_create_results(self, update=False, skip_missing=False, analysis_type=
         except NotFoundError:
             results = None
 
-        self.file_resources = self.get_file_resources(update=update, skip_missing=skip_missing, analysis_type=analysis_type)
+        metadata_yaml = os.path.join(
```

Comment:
The way results are managed could be a bit better designed, however if the functionality is there this is a good start.